### PR TITLE
Rename folder

### PR DESCRIPTION
**Pull request by davis278**
Renamed the E3SM Native Grid Test folder and all the relevant files within it, to E3SM DV3D Native Grid Test since there is now an E3SM VCS Native Grid Plot notebook as well. I wanted to differentiate between the DV3D notebook and the VCS notebook.  

In addition to renaming files and folders, I added more info at the beginning of this DV3D notebook and added the "who to contact" if you have questions paragraph at the end.

### CODE REVIEW

File: scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.html [hunk 1]
```diff
@@ -13172,7 +13172,7 @@ <h2 id="To-Interact-with-the-3D-image...">To Interact with the 3D image...<a cla
...
 <p>The data file for this tutorial is "20180129.DECKv1b_piControl.ne30_oEC.edison.cam.h0.0500-01.nc" from the CMIP6 standard resolution (ne30) piControl run.</p>
-<p>The grid file used is "ne30np4_latlon.091226.nc". To download this grid file, go to <a href="https://portal.nersc.gov/project/acme/hdavis/">https://portal.nersc.gov/project/acme/hdavis/</a> and click on the "ne30np4_latlon.091226.nc" file which should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.</p>
+<p>The grid file used is "ne30np4_latlon.091226.nc". To download this grid file, go to <a href="https://portal.nersc.gov/project/acme/hdavis/">https://portal.nersc.gov/project/acme/hdavis/</a> and click on the "ne30np4_latlon.091226.nc" file whcih should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.</p>
```
Comment (downiec) [inline scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.html]: I noticed the 'whcih' was added back in, should remain as 'which' no?

File: scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.ipynb [hunk 2]
```diff
@@ -71,7 +71,7 @@
     "\n",
     "The data file for this tutorial is \"20180129.DECKv1b_piControl.ne30_oEC.edison.cam.h0.0500-01.nc\" from the CMIP6 standard resolution (ne30) piControl run. \n",
     "\n",
-    "The grid file used is \"ne30np4_latlon.091226.nc\". To download this grid file, go to https://portal.nersc.gov/project/acme/hdavis/ and click on the \"ne30np4_latlon.091226.nc\" file which should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.\n",
+    "The grid file used is \"ne30np4_latlon.091226.nc\". To download this grid file, go to https://portal.nersc.gov/project/acme/hdavis/ and click on the \"ne30np4_latlon.091226.nc\" file whcih should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.\n",
```
Comment (downiec) [inline scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.ipynb]: Same spelling error 'whcih' should be 'which'

File: scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.py [hunk 3]
```diff
@@ -48,7 +48,7 @@
 # 
 # The data file for this tutorial is "20180129.DECKv1b_piControl.ne30_oEC.edison.cam.h0.0500-01.nc" from the CMIP6 standard resolution (ne30) piControl run. 
 # 
-# The grid file used is "ne30np4_latlon.091226.nc". To download this grid file, go to https://portal.nersc.gov/project/acme/hdavis/ and click on the "ne30np4_latlon.091226.nc" file which should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.
+# The grid file used is "ne30np4_latlon.091226.nc". To download this grid file, go to https://portal.nersc.gov/project/acme/hdavis/ and click on the "ne30np4_latlon.091226.nc" file whcih should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.
```
Comment (downiec) [inline scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.py]: Change to 'which' from 'whcih'

File: scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.html (hunk 1 above)
Comment (davis278) [reply scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.html]: Yes, it should be fixed - thanks!  I pushed a commit to fix the typo + some other minor edits.

File: scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.ipynb (hunk 2 above)
Comment (davis278) [reply scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.ipynb]: Yes, it should be fixed - thanks!  I pushed a commit to fix the typo + some other minor edits.

File: scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.py (hunk 3 above)
Comment (davis278) [reply scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.py]: Yes, it should be fixed - thanks!  I pushed a commit to fix the typo + some other minor edits.

**Pull request merged**